test/system-core.js

```diff
@@ -282,7 +283,7 @@ describe('Loading Cases', function() {
   loader.instantiate = async function (path) {
     const source = await new Promise((resolve, reject) => fs.readFile(path, (err, source) => err ? reject(err) : resolve(source.toString())));
     global.System = loader;
-    eval(source + '//# sourceURL=' + path);
+    eval(source + '//# sourceURL=' + pathToFileURL(path).href);
```

Comment:
This allow me to set breakpoints in fixtures in VSCode. If you don't feel right. I can revert it.